streams/src/test/java/org/apache/kafka/streams/KafkaStreamsTest.java

```diff
@@ -1089,34 +1091,30 @@ public void shouldTriggerRecordingOfRocksDBMetricsIfRecordingLevelIsDebug() {
 
     @Test
     public void shouldGetClientSupplierFromConfigForConstructor() {
```

Comment:
This and the next test needed an update, because the refactoring break the mocking -- Cf the change in `MockClientSupplier`, too.